app/src/features/surveys/observations/measurements/list/MeasurementsListCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { mdiTrashCanOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import grey from '@mui/material/colors/grey';
+import IconButton from '@mui/material/IconButton';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { CBMeasurementType } from 'interfaces/useCritterApi.interface';
+
+export interface IMeasurementsListCardProps {
+  /**
+   * The measurement to display.
+   *
+   * @type {CBMeasurementType}
+   * @memberof IMeasurementsListCardProps
+   */
+  measurement: CBMeasurementType;
+  /**
+   * Callback fired on remove.
+   *
+   * @memberof IMeasurementsListCardProps
+   */
+  onRemove: () => void;
+}
+
+/**
+ * Renders a single measurement card.
+ *
+ * @param {IMeasurementsListCardProps} props
+ * @return {*}
+ */
+export const MeasurementsListCard = (props: IMeasurementsListCardProps) => {
+  const { onRemove } = props;
+
+  return (
+    <Card
+      component={Stack}
+      variant="outlined"
+      flexDirection="row"
+      alignItems="center"
+      justifyContent="space-between"
+      p={2}
+      sx={{
+        background: grey[100]
+      }}
+      data-testid="measurements-list-item">
+      <Stack gap={0.75} mt={-0.25}>
+        <Box>
+          <Typography variant="body2">
+            <em>{props.measurement.itis_tsn}</em>
+          </Typography>
+          {/* <Typography variant="body2">
```

Comment:
This is code that we do want, that Jer wrote, but which can't work just yet because the response from Critterbase is in flux. Ideally this, or some version of it, will be uncommented soon and the typography above will be removed.